src/types.rs

```diff
@@ -358,9 +369,9 @@ impl Rewrite for ast::WherePredicate {
                                                     .collect());
 
                     if context.config.spaces_within_angle_brackets && lifetime_str.len() > 0 {
```

Comment:
Above, when calculating `used_width`, I think you need to take into account the width of `colon`, rather than a hard-coded number,